### ISSUE TITLE
iND-BiOS.5004.67 Refuses to boot until re-Flashed

### ISSUE THREAD

**Issue by amerika13** [Ryzee119/OpenXenium#14]
I have had this issue a few times now with my Open Xenium chip where I'll reboot my Xbox, which defaults to the XeniumOS boot, and then I'll select iND-BiOS.5004.67 which was previously working and nothing will ever show up on screen and the Xbox will not boot. If I reboot and reflash it, it works fine.

This has happened three times now in a short period of time (one night). I changed to iND-BiOS.5003.67 to see if that made a difference but I haven't used it enough to know if it's something failing with the bios or the chip. I also have XblastOS loaded as an option and it works every time and has never failed.

**Comment by Ryzee119** [comment]
Not really sure why that is. My guess is it's having issues loading the indios.cfg file from c?

**Comment by amerika13** [comment]
That is possible, but it's probably a bug with it is my guess. Due to me being new to the Xbox scene, I wasn't aware of even needing the indbios.cfg quite yet as I have little solid modern info on just about anything Xbox modding-wise. I did configure the cfg file for the iND-BiOS.5003.67 so that I could boot XMBC from where I installed it on my new HDD.

My guess is that 5004.67 was writing a temp default config file to the file system and something during the reboot process would corrupt the file making it unreadable at some point after a certain amount of reboots or if some process was engaged when I rebooted etc. I never did have that issue with all my reboots with 5003.67.

I have not tried 5004.67 since then as 5003.67 is working perfectly for my needs but it is unlikely to be an issue with Open Xenium. I'll go ahead and close this ticket and if I figure anything out in the future, I'll update.

Thank you for your time!
